perfmetrics/__init__.py

```diff
@@ -111,7 +111,7 @@ def __call__(self, f):
         """Decorate a function or method so it can send statistics to statsd.
         """
         func_name = f.__name__
-        func_full_name = '%s.%s' % (f.__module__, func_name)
+        func_full_name = '{}.{}'.format(f.__module__, func_name)
```

Comment:
Could you please revert all these formatting changes? The reason is that the operator version, which doesn't require a method lookup, is twice as fast as `.format()`. Some of these are on critical paths.

CPython 3.7:
```
In [1]: %timeit '%s%s' % ('prefix', 'stat')
269 ns ± 1.24 ns per loop (mean ± std. dev. of 7 runs, 1000000 loops each)

In [2]: %timeit '{}{}'.format('prefix', 'stat')
403 ns ± 2.7 ns per loop (mean ± std. dev. of 7 runs, 1000000 loops each)
```

CPython 2.7:
```
In [2]: %timeit  '%s%s' % ('prefix', 'stat')
10000000 loops, best of 3: 95.9 ns per loop

In [3]: %timeit  '{}{}'.format('prefix', 'stat')
1000000 loops, best of 3: 213 ns per loop
```

Reply:
Reverted!